### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Kareem24**
Build and style the contact form section

### CODE REVIEW

**Review by was-coder — COMMENTED** [review]
Looks Good but it is still not looking as pixel-perfect as the design. please work on that aspect and merge

File: index.html [hunk 1]
```diff
@@ -237,6 +249,44 @@ <h2 class="heading">About Myself</h2>
...
+              maxlength="500"
+              placeholder="Write your message here"
+              required
+            ></textarea>
+            <!-- <input type="text" name="comment" id="" maxlength="500" width="100px " height="200px"> -->
+
+            <button type="submit" class="btn-light submit-btn">Get It Touch</button>
+          </form>
+        </div>
+      </footer>
```
Comment (Lanr3waju) [inline index.html]: I don't think this element should be a footer, it'll be more semantic as its own section

File: index.html [hunk 2]
```diff
@@ -176,10 +182,16 @@ <h2 class="heading">About Myself</h2>
...
+            ></a>
+            <a href="www.linkedin.com/in/kareem-horlah" class="link"
+              ><i class="fa-brands fa-linkedin-in"></i
+            ></a>
             <a href="#" class="link"><i class="bx bxl-gmail"></i></a>
-            <a href="#" class="link"><i class="fa-brands fa-github"></i></a>
+            <a href="https://github.com/Kareem24" class="link"
+              ><i class="fa-brands fa-github"></i
+            ></a>
             <a href="#" class="link"><i class="fa-solid fa-hand-peace"></i></a>
```
Comment (Lanr3waju) [inline index.html:195]: Although this is not mandatory, I believe this should be an `Angellist` logo and you can also create an account and add the link here

File: index.html [hunk 3]
```diff
@@ -51,10 +51,16 @@ <h1>I'm Kareem Glad to see you!</h1>
...
```
Comment (Lanr3waju) [inline index.html:64]: Same as below

File: css/main.css [hunk 4]
```diff
@@ -207,3 +202,74 @@ hr {
...
+  padding: 50px 18px;
+}
+
+.contact-form h2 {
+  color: #fff;
+  margin: 30px 0 12px;
+}
+
+.contact-form p {
+  font-weight: 400;
```
Comment (Lanr3waju) [inline css/main.css]: I think the default `font-weight` is 400 i.e. regular, so this is redundant, and it's also repeated in other parts of your `css`

File: css/main.css [hunk 5]
```diff
@@ -207,3 +202,74 @@ hr {
...
+  flex-direction: column;
+  justify-content: center;
+  gap: 20px;
+}
+
+.contact-form .input {
+  border: 1px solid #cfd8dc;
+  background: #fff;
+  border-radius: 8px;
+  font-weight: 400;
```
Comment (Lanr3waju) [inline css/main.css]: Same as above

File: css/main.css [hunk 6]
```diff
@@ -207,3 +202,74 @@ hr {
...
+  font-size: 1.06rem;
+  color: #172b4d;
+  width: 100%;
+  height: 100%;
+  padding: 12px;
+
+  /* text-align: center; */
+}
+
+.contact-form .text-area {
```
Comment (Lanr3waju) [inline css/main.css:254]: The `text-area` `font-family` does not match the other `inputs` `font-family`. Please make it consistent across the form.

**Review by Lanr3waju — COMMENTED** [review]
## Great Job, kindly resolve the mentioned issues and add a **description** to your PR(Important).

File: css/main.css (hunk 4 above)
Comment (Kareem24) [reply css/main.css]: Adjusted sir

Comment (Kareem24) [reply css/main.css]: All is been resolved sir

**Pull request merged**